Vagrantfile.example

```diff
@@ -9,11 +9,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
  # If you'd prefer to pull your boxes from Hashicorp's repository, you can
  # replace the config.vm.box and config.vm.box_url declarations with the line below.
  #
- # config.vm.box = "fedora/25-cloud-base"
- config.vm.box = "f25-cloud-libvirt"
+ # config.vm.box = "fedora/30-cloud-base"
+ config.vm.box = "f30-cloud-libvirt"
  config.vm.box_url = "https://download.fedoraproject.org/pub/fedora/linux/releases"\
-                     "/25/CloudImages/x86_64/images/Fedora-Cloud-Base-Vagrant-25-1"\
-                     ".3.x86_64.vagrant-libvirt.box"
+                     "/30/Cloud/x86_64/images/Fedora-Cloud-Base-Vagrant-30-1"\
```

Comment:
Why not using the latest release(f31) here?

Reply:
Was not available when tested this, but will update

Reply:
Updated.